src/app/admin-page/admin-page.component.html

```diff
@@ -1,11 +1,11 @@
 <app-page>
   <admin
     *ngIf="globalVars.showAdminTools()"
-    class="h-100 center-col-width border-left border-right border-color-grey"
+    class="h-100 center-col-width"
```

Comment:
there was an extra wide border at the top of the admin page that was larger than the rest of the page.